examples/meta/src/distance/manhattan_distance.sg

```diff
@@ -0,0 +1,18 @@
+CSVFile f_feats_train("../../data/fm_train_real.dat")
+CSVFile f_feats_test("../../data/fm_test_real.dat")
+
+#![create_features]
+RealFeatures features_train(f_feats_train)
+RealFeatures features_test(f_feats_test)
+#![create_features]
+
+#![create_distance]
+ManhattanMetric distance(features_train, features_train)
+#![create_distance]
+
+#![train_and_init]
+DoubleMatrix dm_train = distance.get_distance_matrix()
+distance.init(features_train, features_test)
+DoubleMatrix dm_test = distance.get_distance_matrix()
```

Comment:
Can you also show the interface to call the distance() method itself (that returns a float)